dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/GAENDataService.java

```diff
@@ -39,24 +39,26 @@
   /**
    * Returns the maximum id of the stored exposed entries for the given batch.
    *
-   * @param keyDate in milliseconds since Unix epoch (1970-01-01)
-   * @param publishedAfter in milliseconds since Unix epoch
-   * @param publishedUntil in milliseconds since Unix epoch
+   * @param keyDate must be midnight UTC
+   * @param publishedAfter when publication should start
+   * @param publishedUntil last publication
+   * @param now the start of the query
    * @return the maximum id of the stored exposed entries for the given batch
    */
   int getMaxExposedIdForKeyDate(
-      UTCInstant keyDate, UTCInstant publishedAfter, UTCInstant publishedUntil);
+      UTCInstant keyDate, UTCInstant publishedAfter, UTCInstant publishedUntil, UTCInstant now);
 
   /**
    * Returns all exposeed keys for the given batch.
```

Comment:
Documentation: how does this "the given batch" relate to the parameters?

Reply:
Added documentation on how each parameter influences the "batch"